src/main/java/com/amazon/jenkins/ec2fleet/EC2RetentionStrategy.java

```diff
@@ -31,14 +33,20 @@ public class EC2RetentionStrategy extends RetentionStrategy<EC2FleetNodeComputer
     @Override
     public long check(final EC2FleetNodeComputer fc) {
         final AbstractEC2FleetCloud cloud = fc.getCloud();
+        boolean isOrphaned = false;
 
         LOGGER.fine(String.format("Checking if node '%s' is idle ", fc.getName()));
```

Comment:
Can you update this log to something like `Checking if node should be terminated`?